packages/terra-popup/src/_PopupArrow.jsx

```diff
@@ -20,10 +22,22 @@ const propTypes = {
   refCallback: PropTypes.func,
 };
 
-const PopupArrow = ({
-  refCallback,
-  ...customProps
-}) => <div {...customProps} className={cx(['popup-arrow', customProps.className])} ref={refCallback} />;
+const PopupArrow = (props) => {
+  const {
+    refCallback,
+    ...customProps
+  } = props;
+  const theme = React.useContext(ThemeContext);
+  const arrowClass = classNames(cx(
```

Comment:
Nit: This formatting is a little inconsistent with the other PR updates. Generally the cx( and ending param are new-lined.

Reply:
Fixed f3eb3d2